### ISSUE TITLE
Deeplab : COCO learning rate

### ISSUE THREAD

**Issue by edwardyehuang** [tensorflow/models#7163]
Hi

Can you provide some configuration when doing the pretrain on COCO dataset with Xception and mobilenetV2 ? (etc learning rate, batch size)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
